main/res/values/strings.xml

```diff
@@ -410,6 +410,7 @@
     <string name="caches_remove_all_completely">Delete all from device</string>
     <string name="caches_remove_selected_completely">Delete selected from device</string>
     <string name="caches_delete_events">Delete past events</string>
+    <string name="caches_upload_bookmarklist">Upload cache(s) to bookmark list</string>
```

Comment:
As you are inside the list menu, how about only "Upload to bookmark list" or (to make clear its about the website) "Add to website bookmark list"

Reply:
Done